demo-app/README.md

```diff
@@ -1,36 +1,38 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+# Model Home
+
+Model Home is Open Truss's demo application to help showcase how to use OT and help develop the OT libraries. It is a [Next.js](https://nextjs.org/) project originally bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app) and `npm run ot:setup`.
 
 ## Getting Started
 
-First, run the development server:
+Install packages:
 
 ```bash
-npm run dev
-# or
-yarn dev
-# or
-pnpm dev
-# or
-bun dev
+npm install
 ```
 
-Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+Run the development server
 
-You can start editing the page by modifying `app/page.tsx`. The page auto-updates as you edit the file.
+```bash
+npm run dev
```

Comment:
I've just been running `script/server` from the `demo-app/` folder and it covers all use cases I think?

Reply:
Heh I forgot about that script! I'll udpate